dotnet/src/Connectors/Connectors.GoogleVertexAI/Core/Gemini/GeminiContent.cs

```diff
@@ -15,8 +15,7 @@ internal sealed class GeminiContent
     /// Ordered Parts that constitute a single message. Parts may have different MIME types.
     /// </summary>
     [JsonPropertyName("parts")]
-    [JsonRequired]
-    public IList<GeminiPart> Parts { get; set; } = null!;
+    public IList<GeminiPart>? Parts { get; set; }
```

Comment:
Add some tests to ensure when the response comes without parts, it can serialize correctly.

![image](https://github.com/microsoft/semantic-kernel/assets/19890735/c0e78ca1-2f66-4420-91e1-af4a50be4197)

Reply:
done